lib/vrl/stdlib/Cargo.toml

```diff
@@ -35,13 +35,13 @@ regex = { version = "1", optional = true }
 rust_decimal = { version = "1", optional = true }
 serde_json = { version = "1", optional = true }
 sha-1 = { version = "0.10", optional = true }
-sha-2 = { package = "sha2", version = "0.9", optional = true }
-sha-3 = { package = "sha3", version = "0.9", optional = true }
+sha-2 = { package = "sha2", version = "0.10", optional = true }
+sha-3 = { package = "sha3", version = "0.10", optional = true }
 strip-ansi-escapes = { version = "0.1", optional = true }
 syslog_loose = { version = "0.16", optional = true }
 tracing = { version = "0.1", optional = true }
 url = { version = "2", optional = true }
-uuid = { version = "0.8", features = ["v4"], optional = true }
+uuid = { version = "1", features = ["v4"], optional = true }
```

Comment:
I think, in general, I'd prefer the use of at least "1.x" as a rule. I know we don't hold to that everywhere, but maybe we pop it in the style guide.

Reply:
Yeah, that's a good point.